### PR TITLE
Fixes #1391 [WIP]

### LINKED ISSUE #1391: comments on dashboard "activity" section not paginating
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### The problem

I was on [the dashboard](https://publiclab.org/dashboard), and selected only the "comments" under activity to scroll through and view those that have been made recently. 

Clicking the pagination bar on the bottom didn't work and each page of comments ("1-2-3-4") loaded the same comments responses, not going back in the history. 

(additions by @jywarren follow)

### Proposed solution

We need to make the page respond to the "page=2" style changes in the URL when people click the page number buttons. 

So to start with, comments are fetched on this line of the `home_controller.rb`:

https://github.com/publiclab/plots2/blob/master/app/controllers/home_controller.rb#L88

See how `.limit(20)` appears below, limiting the returned comments to 20? We should change that to respond to the "page number" which gets added into the URL, just like is done on this line:

https://github.com/publiclab/plots2/blob/master/app/controllers/home_controller.rb#L61

Changing the `.limit(20)` to: `.page(params[:page])` should get the comments paginating as well. 

## Steps to Fix

- [x] claim this issue with a comment here, below, and ask any clarifying questions you need
- [x] set up a repository locally following the README instructions, and make sure that all tests pass
- [x] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) or go to [the chatroom](http://publiclab.org/chat) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

### PR DESCRIPTION
**Pull request by SwordShieldMouse**
Fixes #1391
 
* [ ] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by SwordShieldMouse** [comment]
@jywarren I've tried to fix the issue. When running the rake tests, the last notification I get is 
~~~~
0.68 tests/s, 2.00 assertions/s
Running jasmine tests headlessly
rake aborted!
Don't know how to build task 'spec:javascript' (see --tasks)
~~~~

Is that bad?

File: Gemfile.lock [hunk 1]
```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```
Comment (jywarren) [inline Gemfile.lock]: Hi, thanks for your contribution! Looking over it now. First, do you think you could remove Gemfile.lock from your pull request?

**Comment by jywarren** [comment]
I'm not 100% sure what your test error is, but we can look at the error output in the Travis tests, below. 

I looked and saw: 

`Gem::RemoteFetcher::FetchError: Errno::ECONNRESET: Connection reset by peer - SSL_connect (https://rubygems.org/gems/sass-3.4.22.gem)`

Sometimes tests fail due to slow or faulty response from a remote server. I'm trying to restart the tests and maybe it'll run smoothly this time. You can do this yourself by closing and reopening the PR. Let's see what happens!

File: Gemfile.lock (hunk 1 above)
Comment (SwordShieldMouse) [inline Gemfile.lock]: Sure! Sorry, but how might I go about doing that? Would I have to revert all the changes in Gemfile.lock then recommit and resubmit the pull request?

**Comment by jywarren** [comment]
See? looks great! If you could just remove your changes to Gemfile.lock we can merge. Thanks a million!

File: Gemfile.lock (hunk 1 above)
Comment (jywarren) [inline Gemfile.lock]: Oh i see, sorry, missed this - you can revert a file like this:https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

**Comment by SwordShieldMouse** [comment]
Ok, I removed Gemfile.lock from the PR!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks!

**Pull request merged**